tests/test_wasm_builtin_simd.cpp

```diff
@@ -359,9 +359,15 @@ i8x16 TESTFN i8x16_sub_saturate_s(i8x16 x, i8x16 y) {
 i8x16 TESTFN i8x16_sub_saturate_u(i8x16 x, i8x16 y) {
   return __builtin_wasm_sub_saturate_u_i8x16(x, y);
 }
+
+#ifdef __wasm_unimplemented_simd128__
```

Comment:
where/when would that be defined?

Reply:
This is defined by clang when the `unimplemented-simd128` target feature is enabled. This instruction was removed from the spec proposal, so recent V8 versions refuse to accept it unless an extra flag is passed.